graphbolt/include/graphbolt/utils.h

```diff
@@ -0,0 +1,76 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file graphbolt/utils.h
+ * @brief Header file of utils in graphbolt.
+ */
+#ifndef GRAPHBOLT_UTILS_H_
+#define GRAPHBOLT_UTILS_H_
+
+#include <omp.h>
+
+namespace graphbolt {
+namespace utils {
```

Comment:
no need to create a new namespace, I think all the utils here are still for sampling.

Reply:
I'd do:

include/graphbolt/rangedpickers.h
include/graphbolt/rangedpickers.cc

to host all rangedpickers.

Reply:
Make sense, changed for fix.